Makefile

```diff
@@ -61,6 +61,17 @@ proto-format:
 build-docker:
 	$(DOCKER) build -t celestiaorg/celestia-app -f docker/Dockerfile .
 
+fmt:
+	@go mod tidy -compat=1.17
+	@golangci-lint run --fix
```

Comment:
do we need to call `gofumpt -w` here?

I know it gets run with golangci-lint, but it at least isn't actually fixing the code for me. perhaps I have it configured wrong?